javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,161 @@
+/*
+***************************************************************************************************************************************
+                                                         Voice assistant     
+***************************************************************************************************************************************
+*/
+let userName = "";
+let todolist = [];
+
+function getReply(command) {
+  let tempArray = command.split(" ");
+
+  //get name and reply with the user's name
+  if (command.toLowerCase().includes("hello")) {
+    //make tempArray and pick up the name
+    userName = tempArray[tempArray.length - 1];
+    console.log(`Nice to meet you ${userName}`);
+  }
+
+  //to differenciate the question, using includes what(but then should only asked "what is my name")
+  else if (command.toLowerCase().includes("my name")) {
+    if (userName != "") {
```

Comment:
Try not to use `!=` ever. Even if the two sides of this are of different data types then use some parser, but always use `!==` instead.